docker/nginx/conf.d/default.conf

```diff
@@ -22,10 +22,10 @@
 # which is expected to be running on port 5000/5001.
 
 upstream ui_server {
-  server localhost:5000 fail_timeout=0;
+  server web-ui:5000 fail_timeout=0;
```

Comment:
It is not working on deployment:
`nginx: [emerg] host not found in upstream "web-ui:5000" in /etc/nginx/conf.d/default.conf:25`

Reply:
You probably have a different name for the ui in your production `docker-compose.yml` file.
We have to use the same name for this container. Either you rename it `web-ui` in your production `docker-compose.yml` file or I rename it by those you choose.
What's your preference?

PS: Another solution is to put always same port on localhost (for example 5000) but it differs from Docker best practices…

Reply:
I changed the name, still not working!

Reply:
The port is fixed to 5000 in `docker/nginx/conf.d/default.conf `, so whats the problem ?

Reply:
The problem is that in Docker you will have 2 ports: those inside the container (here 5000) and the port outside the container (for example 32774).

It's adviced to use `web-api` as link to other containers. And `5000` as the used port.
32774 is not known by our scripts. But `web-api` (keyword for our container) and `5000` (internal port) are.

We should be careful to not use `5000:5000` in docker-compose because it block 5000 port. Which is incoherent with docker usage. Docker is built to permit to launch multiple times a container (with multiple random ports).

We will find a solution Monday ;)

Reply:
Didi you?

Reply:
We did!

Reply:
I do not understand why you have removed the docker-compose.full. I do not think that it reduce the image size and brake the Invenio choice. What is the advantage of doing this?

Reply:
I do it for 2 reasons:
1. first we talk about the usage in daily meeting and I understand that in RERO-ils you will only use the Dockerfile and files located in **docker** directory for production use. As a result, I consider the `docker-compose.full.yml` file remains for users that want to discover RERO-ils or to launch it in production.
2. every day usage: as a `docker-compose.yml` file existe, you always have collision with the other name `docker-compose.full.yml` in command line. Try it once or twice. More and you become crazy. It's more readable and more understable a `prod.yml` file.

I understand that we break the Invenio choice, but isn't that already done by all changes RERO made in docker directory and all Dockerfiles?
Moreover Invenio can do mistakes, as https://github.com/rero/rero-ils/issues/713 shows it.